python/src/main/python/drivers/runspv.py

```diff
@@ -921,12 +972,16 @@ def run_image_amber(
             cmd.append(png_image)
         cmd.append(amberscript_file)
         status = 'SUCCESS'
-        try:
-            subprocess_helper(cmd, timeout=TIMEOUT_RUN, verbose=True)
-        except subprocess.TimeoutExpired:
-            status = 'TIMEOUT'
-        except subprocess.CalledProcessError:
-            status = 'CRASH'
+
+        if 'catchsegv' in cmd[0]:
```

Comment:
I would prefer making `maybe_add_catchsegv` return a bool and then checking this here

Reply:
Fixed